packages/manager/cypress/e2e/core/domains/smoke-create-domain-records.spec.ts

```diff
@@ -3,21 +3,66 @@ import { createDomain } from 'support/api/domains';
 import { interceptCreateDomainRecord } from 'support/intercepts/domains';
 import { createDomainRecords } from 'support/constants/domains';
 
+const createCaaRecord = (
+  name: string,
+  tag: string,
+  value: string,
+  ttl: string
+) => {
+  cy.findByText('Add a CAA Record').click();
+
+  // Fill in the form fields
+  cy.get('input#name').type(name);
+  cy.get('[data-qa-autocomplete="Tag"]').click();
+  cy.get('li').contains(tag).click();
+  cy.get('input#value').type(value);
+  cy.get('[data-qa-autocomplete="TTL"]').click();
+  cy.get('li').contains(ttl).click();
+
+  // Save the record
+  cy.get('button[data-testid="save"]').click();
+  cy.wait('@apiCreateRecord');
+};
+
+// Reusable function to edit a CAA record
+const editCaaRecord = (name: string, newValue: string) => {
+  cy.get(`#action-menu-for-record-${name}-button`).click();
+  cy.get('li').contains('Edit').click();
+
+  // Edit the value field
+  cy.get('input#value').clear().type(newValue);
+  cy.get('button[data-testid="save"]').click();
+};
+
+// Reusable function to verify record details in the table
+const verifyRecordInTable = (
+  name: string,
+  tag: string,
+  value: string,
+  ttl: string
+) => {
+  cy.get('[aria-label="List of Domains CAA Record"]') // Target table by aria-label
+    .should('contain', name)
+    .and('contain', tag)
+    .and('contain', value)
+    .and('contain', ttl);
+};
+
 authenticate();
+
```

Comment:
Since we're creating real domain records each run, let's add some clean up in a `before` hook to help prevent resources from accumulating on our CI accounts.

```suggestion
// NOTE: Import `cleanUp` via `import { cleanUp } from 'support/util/cleanup';`
before(() => {
  cleanUp('domains');
});
```